WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -126,6 +141,50 @@ struct ThemesPreviewView: View {
             }
             .navigationBarTitleDisplayMode(.inline)
         }
+        .task { await viewModel.fetchPages() }
+        .sheet(isPresented: $showPagesMenu) { pagesListSheet }
```

Comment:
We should set the `presentationDetent` if iOS 16 or above is available: 

```suggestion
        .sheet(isPresented: $showPagesMenu) {
            if #available(iOS 16, *) {
                pagesListSheet
                    .presentationDetents([.medium, .large])
            } else {
                pagesListSheet
            }
        }
```